frontend/src/charts/choroplethMap/colorSchemes.ts

```diff
@@ -9,73 +10,98 @@ import type {
 
 const { altGrey: ALT_GREY, white: WHITE } = het
 
-const interpolateDarkGreen = d3.piecewise(d3.interpolateRgb.gamma(2.2), [
-  het.mapDarkest,
-  het.mapDarker,
-  het.mapDark,
-  het.mapLight,
-  het.mapLighter,
-  het.mapLightest,
-  het.mapLightZero,
-])
-
-const interpolatePlasma = d3.piecewise(d3.interpolateRgb.gamma(2.2), [
-  het.mapWomenDarkZero,
-  het.mapWomenDarkest,
-  het.mapWomenDarker,
-  het.mapWomenDark,
-  het.mapWomenLight,
-  het.mapWomenLighter,
-  het.mapWomenLightest,
-  het.mapWomenLightZero,
-])
-
-const interpolatedarkRed = d3.piecewise(d3.interpolateRgb.gamma(2.2), [
-  het.mapYouthDarkZero,
-  het.mapYouthDarkest,
-  het.mapYouthDarker,
-  het.mapYouthDark,
-  het.mapYouthMid,
-  het.mapYouthLight,
-  het.mapYouthLighter,
-  het.mapYouthLightest,
-  het.mapYouthLightZero,
-])
-
-const D3_MAP_SCHEMES: Partial<Record<ColorScheme, (t: number) => string>> = {
-  darkgreen: interpolateDarkGreen,
-  plasma: interpolatePlasma,
-  inferno: d3.interpolateInferno,
-  viridis: d3.interpolateViridis,
-  greenblue: d3.interpolateGnBu,
-  darkred: interpolatedarkRed,
+const COLOR_SCHEMES: Record<ColorScheme, string[]> = {
+  darkgreen: [
+    het.mapDarker,
+    het.mapDark,
+    het.mapMid,
+    het.mapLight,
+    het.mapLighter,
+    het.mapLightest,
+  ],
+  plasma: [
+    het.mapWomenDarker,
+    het.mapWomenDark,
+    het.mapWomenMid,
+    het.mapWomenLight,
+    het.mapWomenLighter,
+    het.mapWomenLightest,
+  ],
+  inferno: [
+    het.mapMenDarker,
+    het.mapMenDark,
+    het.mapMenMid,
+    het.mapMenLight,
+    het.mapMenLighter,
+    het.mapMenLighest,
+  ],
+  viridis: [
+    het.mapMedicareDarkest,
+    het.mapMedicareDark,
+    het.mapMedicareMid,
+    het.mapMedicareLight,
+    het.mapMedicareLighter,
+    het.mapMedicareEvenLighter,
+    het.mapMedicareLightest,
+  ],
+  greenblue: [
+    het.unknownMapLeast,
+    het.unknownMapLesser,
+    het.unknownMapLess,
+    het.unknownMapMid,
+    het.unknownMapMore,
+    het.unknownMapEvenMore,
+    het.unknownMapMost,
+  ],
+  darkred: [
+    het.mapYouthDarkZero,
+    het.mapYouthDarkest,
+    het.mapYouthDarker,
+    het.mapYouthDark,
+    het.mapYouthMid,
+    het.mapYouthLight,
+    het.mapYouthLighter,
+    het.mapYouthLightest,
+    het.mapYouthLightZero,
+  ],
 }
 
+const D3_MAP_SCHEMES: Record<ColorScheme, (t: number) => string> =
```

Comment:
I think we should rename this variable or at least at a comment; it's a little confusing that "scheme" above means an array of colors, but here scheme is like, a mapping to an interpolated function I guess? it's a little hard to tell what's going on here, so if we can clarify with var names and comments I think that'll help future us :)